tests/data/README.md

```diff
@@ -4,11 +4,11 @@ This directory provides test data to demonstrate functionality.
 
 ## Sources
 
-### `ne_110m_lakes.geojson`
+### `ne_110m_lakes.geojson / tiles/ne_110m_lakes`
```

Comment:
what about `and` rather `/`? At first look, the whole title seems to be a path in the filesystem

Reply:
Good point; updated.